sklearn/utils/extmath.py

```diff
@@ -332,6 +333,12 @@ def randomized_svd(
             the value of `random_state` explicitly to suppress the deprecation
             warning.
 
+    lapack_driver : str, {'gesdd', 'gesvd'}, default='gesdd'
```

Comment:
```suggestion
    lapack_driver : {"gesdd", "gesvd"}, default="gesdd"
```

Reply:
We should probably rename this to `lapack_svd_driver` just in case we later want to also make it possible to change the qr driver of  the call to`randomized_range_finder` (with one of ``dgeqrf``, ``zgeqrf``,
    ``dorgqr``, and ``zungqr``) for instance.